ParaViewPlugins/interactiveSkeleton/Plugin/Filters/CMakeLists.txt

```diff
@@ -19,6 +19,6 @@ find_package (Eigen3 3.3 REQUIRED NO_MODULE)
 vtk_module_link(interactiveSkeletonFilters
   PUBLIC igl::core igl::common Eigen3::Eigen ${EMBREE_LIBRARIES}
 )
-vtk_module_include(interactiveSkeletonFilters PUBLIC ${EMBREE_INCLUDES} "/Users/coreylee/Git/libigl_Test/libigl/external/embree/include" "/Users/coreylee/Git/libigl_Test/build/include")
-
+# vtk_module_include(interactiveSkeletonFilters PUBLIC ${EMBREE_INCLUDES} "/Users/coreylee/Git/libigl_Test/libigl/external/embree/include" "/Users/coreylee/Git/libigl_Test/build/include")
+vtk_module_include(interactiveSkeletonFilters PUBLIC ${EMBREE_INCLUDES} "/users/jeffhadley/Builds/src/libigl/include/igl/embree" "/users/jeffhadley/Builds/src/libigl/include") 
```

Comment:
can you test removing this line? I'm not entirely sure its needed

Reply:
for background, Embree is a raytracing library and I was experimenting with using a raytracing algorithm for automatic skeleton skinning weights, but I never could get it working.

Reply:
It still works without it in there. Just pushed the changes.

Reply:
You can delete that line. And remove the Embree stuff

Reply:
Should all be cleaned up now.